new_platforms/scripts/Prepare-ForNugetPack.ps1

```diff
@@ -0,0 +1,93 @@
+# Data
+$Leaves = @(
+    "nuget\lib\native\v141\sgx\hw\x86\Debug",
+    "nuget\lib\native\v141\sgx\hw\x64\Debug",
+
+    "nuget\lib\native\v141\sgx\sim\x86\Debug",
+    "nuget\lib\native\v141\sgx\sim\x64\Debug",
+
+    "nuget\lib\native\v141\tz\hw\arm\Debug",
+    "nuget\lib\native\v141\tz\sim\x86\Debug",
+
+    "nuget\build\native\include"
+)
+
+$EnclaveLibraries = @(
+    "oeenclave",
+    "oesocket_enc",
+    "oestdio_enc"
+)
+
+$HostLibraries = @(
+    "oehost",
+    "oesocket_host",
+    "oestdio_host"
+)
+
+$OPTEESimLibraries = @(
+    "oeenclave_opteesim",
+    "oehost_opteesim"
+)
+
+$Extensions = @(
+    "lib",
+    "pdb"
+)
+
```

Comment:
the package seems to be missing exe's needed to build.  Namely oeedger8r.exe and I think oesign.exe.

Reply:
Added.